src/app/App.tsx

```diff
@@ -0,0 +1,18 @@
+import { routeConfig } from './providers/router/routeConfig/appRouteConfig';
+import { AppRouter } from 'app/providers/router';
```

Comment:
лучше использовать какой-то один подход - или везде относительный путь, или везде абсолютный. но тк тут импорт в рамках модуля, можно использовать относитнльный
```suggestion
import { routeConfig } from './providers/router/routeConfig/appRouteConfig';
import { AppRouter } from './providers/router';
```